platforms/jvm/language-groovy/build.gradle.kts

```diff
@@ -71,6 +71,12 @@ dependencies {
     integTestDistributionRuntimeOnly(project(":distributions-jvm"))
 }
 
+tasks.withType<Test>().configureEach {
+    if (!javaVersion.isJava9Compatible) {
+        classpath += javaLauncher.get().metadata.installationPath.files("lib/tools.jar")
+    }
+}
```

Comment:
ℹ️ This existed already in the `language-java` project, and now caused a difference. The spec selection (through `JdkJavaCompiler.canBeUsed()` needs the `tools.jar` on the classpath to make the correct decision.

This is the default when we run the daemon itself, but this unit test didn't have that setup.